include/SFML/Window/Event.hpp

```diff
@@ -267,31 +267,37 @@ class SFML_WINDOW_API Event
     };
 
     ////////////////////////////////////////////////////////////
-    /// \brief Construct from a given sf::Event subtype
+    /// \brief Construct from a given `sf::Event` subtype
     ///
-    /// \param t Event subtype
+    /// \tparam TEventSubtype Type of event subtype used to construct the event
+    ///
+    /// \param eventSubtype Event subtype used to construct the event
```

Comment:
What about adding the word "instance" to clarify when we are referring to values instead of types?